orisa/orisa.py

```diff
@@ -886,7 +886,7 @@ async def register(self, ctx, type: str = "pc", *, online_id: str = None):
             await reply(
                 ctx,
                 _(
-                    'Invalid registration type "{type}". Use `!ow register` or `!ow register pc` for PC; `!ow register xbox` for XBOX, or `!ow register psn My-Online-Id_1234` for PlayStation.'
```

Comment:
Sneaky… Orisa doesn't judge (I mean, she does, as PC is the default, but no point in rubbing it in ;)

Reply:
None intended. I think everyone can take the joke. I thought it fit into the scheme of things and language used elsewhere. I tried to loosen up the rigid parts to match the general tone too.